metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/publish/statistics/MeterDetailStatisticsModel.java

```diff
@@ -0,0 +1,204 @@
+package org.apache.servicecomb.metrics.core.publish.statistics;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.servicecomb.metrics.core.meter.invocation.MeterInvocationConst;
+import org.apache.servicecomb.metrics.core.publish.model.invocation.OperationPerf;
+import org.apache.servicecomb.metrics.core.publish.model.invocation.PerfInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MeterDetailStatisticsModel {
```

Comment:
why need this model?

enum the stages, just like the old logic, seems simpler?

Reply:
well. the data structure is different with  before.
for example:

old:
rest.200:
  xx  xxx xx xx  sayHi
  xxx xx xx xx   sayHello
rest.400:
  xx  xxx xx xx  sayHi
  xxx xx xx xx   sayHello

now: 
sayHi:
   rest200:
      xx xx xxx
   rest400:
      xx xx xxx
sayHello:
   rest200:
      xx xx xxx
   rest400:
      xx xx xxx
  Hence, I have to load all  data, And then restructure it